small-talk/db/registerToDB.js

```diff
@@ -5,18 +5,29 @@ import bcrypt from "bcryptjs";
 export default async function handleRegister(username, password, firstname, lastname, dob){ // function that sends the credentials to the database
   const mongoboi = new Mongoboi(uri, "Users");
   const date = new Date(dob).getTime() / 1000; // return the inputted date as a Unix timestamp
-  await mongoboi.connect();
-
-  const hashedpass = await bcrypt.hash(password, 10);
-
+  //await mongoboi.connect();
+  
   const newUser = { // schema for holding the values into certain fields for database organization
     username: username,
-    password: hashedpass,
     firstname: firstname,
     lastname: lastname,
     dob: date,
   };
 
+  bcrypt.genSalt(10, async function(saltError, salt){ // generate a salt for password
```

Comment:
I just have a small critique on the readability of this fragment. This function nested inside makes it a little tough to read but I understand if that is the way it must be.